cmd/agent/app/flags_test.go

```diff
@@ -51,3 +53,39 @@ func TestBindFlags(t *testing.T) {
 	assert.Equal(t, 42, b.Processors[2].Server.QueueSize)
 	assert.Equal(t, 42, b.Processors[2].Workers)
 }
+
+func TestExposeTuningOptions(t *testing.T) {
+	v := viper.New()
+	b := &Builder{}
+	command := cobra.Command{}
+	flags := &flag.FlagSet{}
+	AddFlags(flags)
+	command.PersistentFlags().AddGoFlagSet(flags)
+	v.BindPFlags(command.PersistentFlags())
+
+	err := command.ParseFlags([]string{
+		"--http-server.host-port=:8080",
+		"--processor.jaeger-binary.server-host-port=:1111",
+		"--processor.jaeger-binary.server-max-packet-size=4242",
+		"--processor.jaeger-binary.server-queue-size=42",
+		"--processor.jaeger-binary.workers=42",
```

Comment:
Quite a small thing, but could you change this to be different than the `server-queue-size`? Same as some other test in this PR.

Reply:
As I see ```--span-storage.type``` is deprecated in [1.19](https://www.jaegertracing.io/docs/1.19/cli/#jaeger-collector).

Reply:
It is since like forever, as we need it in order to setup the storage-related flags, so, people have to use an env var. But my point is that we could expose what's the storage type that is being used by the instance, and I think it is indeed stored in Viper's key for this flag.